### DISCUSSION TITLE
Jandy RS-2/6 Dual w/ locked up ONETouch....may be wiring issue.

### DISCUSSION THREAD

**Discussion by Bugman1400** [aqualinkd/AqualinkD#256] [General]
I installed the "Trust Everything" version of AqualinkD. Ran the serial logger and got the respective Suggested aqualinkd.conf values. The Web GUI seems to work good, but my local ONETouch panel is frozen and does not accept input from the front buttons. When I stop the aqualinkd service, the ONETouch is still frozen; however, when I unplug the 485toUSB converter from the RPi the ONETouch returns to normal. I believe I only have the BLACK and YELLOW wires connected to the 485 bus. 
Any suggestions?

**Comment by sfeakes** [comment]
If AqualinkD is working you have the correct wires connected.  It sounds like you need to change where on the RS485 bus you are connecting the pi or the one touch.  Since this is unique to every setup, you may have to play around to get it all working. The wiki explains how the bus works, you should read and understand that then move the connections around.

**Comment by Bugman1400** [comment]
UPDATE: I'm a dummy. Not sure why I thought BLACK & YELLOW (Data+ & Data-) in the AqualinkRS panel was equal to BLACK & YELLOW on the RS485toUSB converter. IT IS NOT. There was no pinout diagram that came with RS485toUSB converter, but I looked it up for a similar product and found out that the BLACK wire was actually GROUND.
All comms seem to be working correctly. Yay!
Thank you!
